node/src/effect.rs

```diff
@@ -1524,6 +1524,6 @@ impl<REv> EffectBuilder<REv> {
 #[macro_export]
 macro_rules! fatal {
     ($effect_builder:expr, $($arg:tt)*) => {
-        $effect_builder.fatal(file!(), line!(), format_args!($($arg)*).to_string()).ignore()
+        $effect_builder.fatal(file!(), line!(), format_args!($($arg)*).to_string())
```

Comment:
would we ever want to create an event that follows the call to `fatal!`? I thought that it's supposed to represent a stopping condition for the whole app.

Reply:
No, but if it contains the `ignore` it can't be chained with other futures.

Reply:
That's correct. `.ignore()` turns it from a generic future into an `Effect`. Not having `ignore` called right away makes composition a lot easier.